ui/src/themes/useCurrentTheme.js

```diff
@@ -0,0 +1,15 @@
+import { useSelector } from 'react-redux'
+import useMediaQuery from '@material-ui/core/useMediaQuery'
+import LightTheme from './light'
+import DarkTheme from './dark'
+import { AUTO_THEME_ID } from '../consts'
+
+export default () => {
+  const prefersLightMode = useMediaQuery('(prefers-color-scheme: light)')
+  return useSelector((state) => {
+    if (state.theme === AUTO_THEME_ID) {
+      return prefersLightMode ? LightTheme : DarkTheme
+    }
+    return state.theme === 'LightTheme' ? LightTheme : DarkTheme
```

Comment:
After rebasing with changes from master, that includes a new theme "Spotify-ish", this is not the right way to select themes.

Reply:
Oh, thanks for pointing that out! I have updated with master and changed the line too. Tested it on my machine. Please take a look.